### ISSUE TITLE
does nghttp2 supports http 1.1 protocol ?

### ISSUE THREAD

**Issue by mamond** [nghttp2/nghttp2#585]
does nghttp2 supports http 1.1 protocol ? I want to write a common client built on top of mghttp2  so that i can send http1.1 and http2 request.

**Comment by tatsuhiro-t** [comment]
No.  libnghttp2 only understands HTTP/2.

**Comment by terrywh** [comment]
does nghttp2  supports http 1.1 protocol on the server side ? (for client/browser that does not support http/2)

**Comment by tatsuhiro-t** [comment]
No. libnghttp2 only understands HTTP/2.  You have to write HTTP/1 code, for example, using nodejs/http-parser.

**Comment by nqf** [comment]
https://github.com/fantasy-peak/simple_http
